book/src/reference.md

```diff
@@ -75,6 +75,31 @@ Subfields:
     - For the `html` renderer, the default value is `html`.
     - For all other renderers, the default value is `preserve`.
 
+### `custom`
+
+Optional.
+
+Additional type of block to support.
+You must run `mdbook-admonish generate-custom` after updating these values, to generate the correct styles.
+
+Add blocks using TOML's [Array of Tables](https://toml.io/en/v1.0.0#array-of-tables) notation:
+
+```toml
+[[preprocessor.admonish.custom]]
+directive = "expensive"
+icon = "./money-bag.svg"
+color = "#24ab38"
+aliases = ["money", "cash", "budget"]
+```
+
+Subfields:
+
+- `directive`: The keyword to use this type of block.
+- `icon`: A filepath relative to the book root to load an SVG icon from.
+- `color`: An RGB hex encoded color to use for the icon.
+- `aliases` (optional): One or more alternative directives to use this block.
```

Comment:
how do aliases interact with custom default titles?

Reply:
i had worked around this problem originally by having aliases just not exist and instead duplicating the custom directives for each alias

Reply:
I'll document - the same title would apply to all aliases. Basically, if you want different titles, just add two custom blocks

Reply:
:+1: